docs/_config.yml

```diff
@@ -353,6 +353,12 @@ ja:
   chapters:
     - path: ja/week07/07.md
       sections:
-        - path: ja/week07/07-1.md
+        - path: ja/week07/07-1.md<<<<<<< master
```

Comment:
What is this `<<<<<<< master`?

Reply:
Oops, this shouldn't be there.. 
Okay.. it's been removed